draft-ietf-rats-architecture.md

```diff
@@ -78,6 +78,14 @@ informative:
     author:
       org: "Trusted Computing Group"
     target: "https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p62_Part1_Architecture_7july2020.pdf"
+  WebAuthN:
+    title: Web Authentication: An API for accessing Public Key Credentials
+    author: The W3C
+    target: https://www.w3.org/TR/webauthn-1/ 
+  CTAP:
+    title: Client to Authenticator Protocol
+    author: The FIDO Alliance
```

Comment:
```suggestion
    author: FIDO Alliance
```
(The copyright at the bottom of the linked URL just says "FIDO Alliance", no "The" in front)